exercises/08_futures/03_runtime/src/lib.rs

```diff
@@ -24,7 +24,7 @@ where
 {
     loop {
         let (mut socket, _) = listener.accept().await.unwrap();
-        let (mut reader, mut writer) = socket.split();
+        let (mut _reader, mut writer) = socket.split();
```

Comment:
```suggestion
        let (_reader, mut writer) = socket.split();
```
We can alos remove the `mut` here